Makefile

```diff
@@ -1,6 +1,7 @@
 SHELL := /bin/bash
 CURRENT_UID := $(shell id -u)
 CURRENT_GID := $(shell id -g)
+newsblur := $(shell docker ps -qf "name=newsblur_web")
```

Comment:
Rename to a description variable. A `newsblur` variable in the Makefile isn't as description as newsblur_docker_pid or whatever this command does.

Reply:
I see its just moved from below, but let's rename anyway